assets/scripts/streets/__tests__/StreetMetaWidthContainer.test.js

```diff
@@ -29,10 +30,8 @@ describe('StreetMetaWidthContainer', () => {
         updateStreetWidth={updateStreetWidth}
       />
     )
-    fireEvent.click(screen.getByTitle('Change width of the street'))
-    fireEvent.change(screen.getByRole('listbox'), {
-      target: { value: changeValue }
-    })
+    userEvent.click(screen.getByTitle('Change width of the street'))
+    userEvent.type(screen.getByRole('listbox'), changeValue)
```

Comment:
`userEvent.selectOptions` should work better for a select dropdown, would you agree?

https://github.com/testing-library/user-event#selectoptionselement-values

Reply:
I'm not 100% which part this is testing. Is it after you hover over a segment and can change the width by either typing a new value or hitting the plus and minus buttons?

In which case I'd assume we'd follow user behavior. So if were testing a user typing width of 10 thats what I'd test. 

UPDATE: ok, now I can see that this is for updating the length of the street overall. That wasn't clear to me from the code and, to be honest, I never  noticed that functionality existed before. But yes, would make sense to be a dropdown select.